view/adminhtml/web/css/onboard-styles.css

```diff
@@ -40,5 +59,7 @@
             margin-right: .75em;
         }
     }
+
+    .banner-sidebar > * + * {margin-top: .5em;}
```

Comment:
The `>` is for direct children elements and the `+` selects sibling elements. Basically it selects the plain `a` link below the button since that is conditionally there depending on if we're seeing the 'in progress' or 'completed' state of the banner. 

If this selector is too abstract I can change it to be more explicit.

Reply:
I don't think I've seen `*` used in CSS selectors before.

Reply:
Oh! Yeah the `*` basically denotes "any element"

Reply:
Updated this to be `.banner-sidebar > *:not(:first-child) {margin-top: .5em;}` to be more readable